README.md

```diff
@@ -13,6 +13,7 @@ Image are hosted on DockerHub: https://hub.docker.com/u/pangeo
 | base-image      | Foundational Dockerfile for builds            | ![](https://img.shields.io/docker/image-size/pangeo/base-image?sort=date) | ![](https://img.shields.io/docker/pulls/pangeo/base-image?sort=date)
 | [base-notebook](base-notebook/packages.txt) | minimally functional image for pangeo hubs | ![](https://img.shields.io/docker/image-size/pangeo/base-notebook?sort=date) | ![](https://img.shields.io/docker/pulls/pangeo/base-notebook?sort=date)
 | [pangeo-notebook](pangeo-notebook/packages.txt) | above + core earth science analysis packages | ![](https://img.shields.io/docker/image-size/pangeo/pangeo-notebook?sort=date) | ![](https://img.shields.io/docker/pulls/pangeo/pangeo-notebook?sort=date)
+| [pytorch-notebook](pytorch-notebook/packages.txt) | above + GPU-enabled pytorch | ![](https://img.shields.io/docker/image-size/pangeo/pytorch-notebook?sort=date) | ![](https://img.shields.io/docker/pulls/pangeo/pytorch-notebook?sort=date)
 | [ml-notebook](ml-notebook/packages.txt) | above + GPU-enabled tensorflow2 | ![](https://img.shields.io/docker/image-size/pangeo/ml-notebook?sort=date) | ![](https://img.shields.io/docker/pulls/pangeo/ml-notebook?sort=date)
```

Comment:
Instead of saying 'above', maybe better to explicitly say that `pytorch-notebook` extends `pangeo-notebook`? I got a bit confused by this...
```suggestion
| [pangeo-notebook](pangeo-notebook/packages.txt) | base-notebook + core earth science analysis packages | ![](https://img.shields.io/docker/image-size/pangeo/pangeo-notebook?sort=date) | ![](https://img.shields.io/docker/pulls/pangeo/pangeo-notebook?sort=date)
| [pytorch-notebook](pytorch-notebook/packages.txt) | pangeo-notebook + GPU-enabled pytorch | ![](https://img.shields.io/docker/image-size/pangeo/pytorch-notebook?sort=date) | ![](https://img.shields.io/docker/pulls/pangeo/pytorch-notebook?sort=date)
| [ml-notebook](ml-notebook/packages.txt) | pangeo-notebook + GPU-enabled tensorflow2 | ![](https://img.shields.io/docker/image-size/pangeo/ml-notebook?sort=date) | ![](https://img.shields.io/docker/pulls/pangeo/ml-notebook?sort=date)
```

Alternatively, we can have a diagram similar to https://jupyter-docker-stacks.readthedocs.io/en/latest/using/selecting.html#image-relationships
![image](https://user-images.githubusercontent.com/23487320/165637274-f761df3a-a9ee-49e8-aa8e-240e4ed65b42.png)

Reply:
"pytorch-notebook extends pangeo-notebook " sounds good.  A diagram would be great, but let's do that later:  If directly in the repository readme can use this https://github.blog/2022-02-14-include-diagrams-markdown-files-mermaid

Reply:
Thanks for the link to that! I hadn't realize GitHub had added this already, previously I was using https://yuml.me but it's much nicer to have an in-built solution. Anyways, I've added the diagram in another commit, looks like this:

![image](https://user-images.githubusercontent.com/23487320/165641952-0fbdd229-2a60-47d7-8e68-3a4b5bc03840.png)